lld/test/wasm/compress-relocs.s

```diff
@@ -0,0 +1,130 @@
+# RUN: llvm-mc -mattr=+reference-types,+exception-handling -filetype=obj -triple=wasm32-unknown-unknown %s -o %t.o
```

Comment:
If we are going to convert this test to assembly (a great idea I think), I would prefer to land the conversion as a separate no-op change, before this PR lands.